src/transformers/models/auto/modeling_auto.py

```diff
@@ -448,6 +448,13 @@
     ]
 )
 
+MODEL_FOR_DEPTH_ESTIMATION_MAPPING_NAMES = OrderedDict(
+    [
+        # Model for depth estimation mapping
+        ("dpt", "DptForDepthEstimation"),
```

Comment:
```suggestion
        ("dpt", "DPTForDepthEstimation"),
```

This is your bug, `Dpt` does not exist, so it loops trying to find this name.